app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,21 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf warrior {self.nickname}."
+                f" {self.nickname} has a hummer"
+                f" of the {self._hummer_level} level"
+                )
```

Comment:
Do not place single parenthesis here and in the other places in `player_info`, place it on the previous line to match the opening one